R/DiseasystoreGoogleCovid19.R

```diff
@@ -136,10 +136,28 @@ google_covid_19_metric <- function(google_pattern, out_name) {
       coll <- checkmate::makeAssertCollection()
       checkmate::assert_date(start_date, lower = as.Date("2020-01-01"), add = coll)
       checkmate::assert_date(end_date,   upper = as.Date("2022-09-15"), add = coll)
+      checkmate::assert(
```

Comment:
Does `dir.exists(source_conn)` (used below) always return `FALSE` for an FTP connection?

I would handle this more explicitly, replacing the assertion with if-elif-else to either set a variable (e.g. `source_type`) to "directory" or "url" and `stop()` if no conditions are met.

Reply:
Updated to be more robust (and to reuse the `checkmate::` checks rather than `dir.exists`)

Adding an `source_type` parameter to control the logic would, I think, mean large changes to the how we handle features in general since the number of arguments to `compute` will then change across `FeatureHandlers`.

Reply:
Ahh, I see. I will keep this in mind.